collector.go

```diff
@@ -27,20 +27,22 @@ const (
 
 var (
 	labelNames = []string{"ip", "host"}
+)
+
+func init() {
+}
 
-	pingResponseSeconds = prometheus.NewHistogramVec(
+func newPingResponseHistogram(buckets []float64) **prometheus.HistogramVec {
```

Comment:
The double pointer is weird here. Simplify to `*prometheus.HistogramVec` and then the body can be:

```
return prometheus.NewHistogramVec(/* ... */)
```